go-controller/pkg/util/multi_network.go

```diff
@@ -59,6 +60,14 @@ func (nInfo *DefaultNetInfo) GetNetworkName() string {
 	return types.DefaultNetworkName
 }
 
+// IsPrimaryNetwork returns if this secondary network is
+// meant to be the primary network for the pod. This cannot
+// be true if IsSecondary() is not true. Hence this is always
+// false for default network.
```

Comment:
Let's directly say from the beginning that this return always false, and then explain why.

Reply:
done.